src/ZXW/utils.jl

```diff
@@ -667,3 +667,14 @@ function substitute_variables!(
     end
     return zxwd
 end
+
+
+function convert_to_zxwd(root::YaoHIR.BlockIR)
+    # FIXME add test
```

Comment:
add tests?

Reply:
Test is in ir.jl in line 57. Forgot to remove the FIXME